src/server/controllers/collectives.js

```diff
@@ -28,3 +29,86 @@ export async function info(req, res, next) {
 
   res.send(response);
 }
+
+const query = `
+query Account($slug: String) {
+  account(slug: $slug) {
+    memberOf(role: HOST) {
+      nodes {
+        createdAt
+        account {
+          ... on Collective {
+            slug
+            name
+            website
+            description
+            isApproved
+            balance
+            updatedAt
+          }
+          members(role: [ADMIN,CONTRIBUTOR] ) {
+            nodes {
+              role
+              account {
+                ... on Individual {
+                  email
+                  name
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+`;
+
+export async function hosted(req, res) {
+  try {
+    const data = await getClient({ version: 'v2' }).request(query, { slug: req.params.slug });
+    const hostedCollectives = get(data, 'account.memberOf.nodes');
+
+    if (isEmpty(hostedCollectives)) {
+      res.send(JSON.stringify([]));
+    }
+
+    const response = [];
+    for (const collective of hostedCollectives) {
+      const { createdAt = null, account } = collective;
+      const result = {
+        ...pick(account, ['name', 'website', 'description', 'balance', 'isApproved']),
+        approvedDate: createdAt,
+        lastActive: get(account, 'updatedAt'),
+        admins: [],
+        totalFinialContributors: 0,
+      };
+
+      const members = get(account, 'members.nodes');
+
+      if (!isEmpty(members)) {
+        members.map(member => {
+          if (member.role === 'ADMIN') {
+            result.admins.push(member.account);
```

Comment:
What's the strategy for exporting admins in CSV ?

Reply:
```js
query 
...
  members(role: ADMIN) {
    nodes {account {... on Individuals {name, email}}}
  }
```
Find the members that are `ADMIN` and fetch required fields `name, email`